Source/FullScreenPokemon.js

```diff
@@ -1034,7 +1034,7 @@ var FullScreenPokemon;
             }
             thing.cycling = true;
             thing.FSP.addStateHistory(thing, "speed", thing.speed);
-            thing.speed = this.MathDecider.compute("speedCycling", thing);
+            thing.speed = thing.FSP.MathDecider.compute("speedCycling", thing);
```

Comment:
You know... I'm not sure where that change came from. I don't recall adding this change or why that was `this` prior to this.